instrumentation/opentelemetry-instrumentation-cherrypy/src/opentelemetry/instrumentation/cherrypy/__init__.py

```diff
@@ -0,0 +1,167 @@
+from logging import getLogger
+from time import time_ns
+from timeit import default_timer
+from typing import Collection
+
+from opentelemetry.util.http import parse_excluded_urls, get_excluded_urls, get_traced_request_attrs
+import cherrypy
+from opentelemetry.instrumentation.cherrypy.package import _instruments
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import _start_internal_or_server_span, extract_attributes_from_object
+import opentelemetry.instrumentation.wsgi as otel_wsgi
+from opentelemetry.instrumentation.propagators import (
+    get_global_response_propagator,
+)
+from opentelemetry import trace, context
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.instrumentation.cherrypy.version import __version__
+from opentelemetry.metrics import get_meter
+
+
+_logger = getLogger(__name__)
+_excluded_urls_from_env = get_excluded_urls("CHERRYPY")
+
+
+class CherryPyInstrumentor(BaseInstrumentor):
+    """An instrumentor for FastAPI
```

Comment:
Change the docstring from FastAPI to CherryPy. Also please add an example how to use it